lib/fingerprint.js

```diff
@@ -68,7 +68,7 @@ Fingerprint.prototype.getDestFilePath = function (relativePath) {
     }
 
     var tmpPath = path.join(this._srcDir, relativePath);
-    var file = fs.readFileSync(tmpPath, { encoding: 'utf8' });
+    var file = fs.readFileSync(tmpPath, { encoding: null });
```

Comment:
Most important part of the change.  Force a buffer back from the `fs.readFileSync(...)` by passing the `null` encoding.